README.md

```diff
@@ -25,39 +22,48 @@ This is the offical documentation for a Yarny Social Media platform project buil
-- 💎 Basic Styling: Tailwind CSS is used as a UI framework to style the components. The UI is clean and responsive.
-- 💎Bonus Features:
-- Comments functionality where users can comment on articles.
-- Authenticated users can like or unlike an article. While unauthenicated users can view likes only
-- Users can see number of views an article has
-- Users can share artilce on Social Media (Facebook, Twitter and Whatsapp) -->
+💎 CRUD Functionality for yarn (post): Users can create and read yarn for now. Each yarn has an a username, content, like yarn button, reyarn, comment buton and a created/updated timestamp. A simple text editor is used for comment . Only a user can comment on a yarn, like yarn and read yarn.
+
+- 💎 Users can Onoboard the Application using user friendly means, Using their prefered email and Password (under the hood, DID which is lengthy to input,control and not memorisable, is store by users, during registration).
+
+- 💎 At login the app only verify the user, email and password.. then All other Security, verification, Storage and retrieval of Data is from the DWN, using the DID of the User.
+
+- 💎 All Data provided at registration are used to initialize the DID and Make Data available in the DWN.
+
+- 💎 Yarny only verify a user email and password, then Pass the DID, to the User under the Hood for DWN calls, no Data is stored or Handled by Yarn.
+
+## Future Features
+
+- 💎 Account Deactivation: if the user deciders to deactivate His/Her account, The DWN holders all there Data, the Account cannot be reactivated, the user can only open another account.
+
+- 💎 Profile Editing: if the user deciders to delete or change any profile data in their DWN.
+
+- 💎 Direct Messaging: if the user deciders to Send a Direct message another user on the platform, they can communicate privately to each other.
+
+- 💎 Analytic: This will provide insight to activities of the user, like the amount of follower, Yarns, Reyarns, Shared Yarns and more.
+
+- 💎 Settings This will be for moderation and control. what to see and not to see
 
 ## Getting Started
 
-To run this project locally, you will need to have Node 19, yarn/npm, and TypeScript installed on your machine. You will also need to be familar with the Web5 documentation [here](https://developer.tbd.website/docs/web5/).
+To run this project locally, you will need to have Node 19 or later, yarn/npm and TypeScript installed on your machine. You will also need to be familar with the Web5 documentation [here](https://developer.tbd.website/docs/web5/).
 
 ## Installation
 
 Clone this repository to your local machine:
 
 ```
 
-git clone https://github.com/todak2000/xxxyyyzzz.git
+https://github.com/todak2000/yarny.git
 
 ```
 
 Navigate to the project directory and install the dependencies:
 
 ```
 
-cd micro-blog
-yarn install next@latest webpack@latest
+cd yarn
```

Comment:
cd yarny not yarn??

Reply:
yes.. yarny is the project directory... yarn is the package...

Reply:
fixed